spec/lib/identity_config_spec.rb

```diff
@@ -29,4 +29,29 @@
       end
     end
   end
+
+  describe 'in_person_outage_message_enabled' do
```

Comment:
added tests for the configs to make sure dates are included when the outage message is enabled. Alert will now not show unless dates are included rather than keeping pages that have alert to not be rendered. Want feedback to this approach.